osu.Game.Rulesets.Osu/Mods/OsuModRepel.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Would be preferable to have all new code have NRT enabled. The problem here is going to be `gameplayClock`, but I think the way to go here is to annotate it as nullable and then add a `Debug.Assert(gameplayClock != null)` in `easeTo()`.